frontend/apps/filemanager/filemanagerbookinfo.lua

```diff
@@ -107,8 +107,7 @@ function BookInfo:show(file, book_props)
             -- for all others than crengine, we seem to get an accurate nb of pages
             pages = document:getPageCount()
         end
-        -- via pcall because picdocument:getProps() always fails (we could
-        -- check document.is_pic, but this way, we'll catch any other error)
+        -- via pcall, even if getProps() shouldn't fail anymore
```

Comment:
No, we shouldn't need to care. But if it shouldn't fail, why use pcall ?
I was tempted to remove all these pcalls, as all getProps() seem to have themselves a pcall for stuff that may fail. But there may be some cases i haven't seen that could cause a crash, and i was a bit frightened to do without the pcall (getProps being now called when a document is open, not just when inspecting its Book information).
So, should i remove the pcall or just the "anymore" ? :)